src/commands/realType.ts

```diff
@@ -52,8 +43,7 @@ export async function realType(text: string, options: RealTypeOptions = {}) {
     }, [] as string[]);
 
   for (const char of chars) {
-    assertChar(char);
```

Comment:
this isn't needed as `getKeyCodeDefinitions` throws on unknown characters anyway